content/en/tracing/profiling/proxy.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Proxy configuration with Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+Profiles collected by the Datadog libraries are sent directly from your running application to Datadog. If your network configuration is restricted for outbound traffic use your proxy to redirect the profiles to Datadog.
+
+If setup, your proxy should point to the following Datadog profiles endpoints:
+
+- If you are on Datadog US site. `https://intake.profile.datadoghq.com`
+- If you are on Datadog EU site. `https://intake.profile.datadoghq.eu`
```

Comment:
```suggestion
```

Summary of offline convo- it's not clear people need these hostnames.

Reply:
+1